### PR TITLE
Add pseudo pip data and fix some parameter pins.

### PR DESCRIPTION
**Pull request by litghost**

### CODE REVIEW

File: fpga_interchange/chip_info.py [hunk 1]
```diff
@@ -670,7 +684,8 @@ def __init__(self):
         self.name = ''
         self.generator = ''
 
-        self.version = 0
+        # Note: Bump this whenever schema changes.
+        self.version = 1
```
Comment (acomodi) [inline fpga_interchange/chip_info.py:688]: What do you mean by bumping exactly? (e.g. increasing by 1, assigning the version of the schema, etc.)

File: fpga_interchange/populate_chip_info.py [hunk 2]
```diff
@@ -208,12 +210,19 @@ def __init__(self, device, tile_type_index, tile_type, cell_bel_mapper,
...
 
-        # Add pips
+        # Add pips, collecting pseudo_pips for later processing.
+        pseudo_pips = []
         for idx, pip in enumerate(tile_type.pips):
-            # TODO: Handle pseudoCells
-            self.add_tile_pip(idx, pip.wire0, pip.wire1)
+            pip_index = self.add_tile_pip(idx, pip.wire0, pip.wire1)
+
+            is_pseudo_cell = pip.which() == 'pseudoCells'
```
Comment (acomodi) [inline fpga_interchange/populate_chip_info.py:218]: What is a pseudoCells here? I see that it is in the fpga-schema, and what I don't fully get is why there are multiple pseudoCells for one pip.

**Review by acomodi — APPROVED** [review]
LGTM

File: fpga_interchange/populate_chip_info.py (hunk 2 above)
Comment (litghost) [reply fpga_interchange/populate_chip_info.py:218]: https://github.com/SymbiFlow/fpga-interchange-schema/pull/18

File: fpga_interchange/chip_info.py (hunk 1 above)
Comment (litghost) [reply fpga_interchange/chip_info.py]: Clarified comments.

**Pull request merged**